### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by pkubica**
removed // from "http:////releases.manageiq.org/manageiq-ovirt-fine-1.qc2"

### CODE REVIEW

**Comment by ovirt-infra** [comment]
Hello contributor, thanks for submitting a PR for this project! 

I am the bot who triggers "standard-CI" builds for this project.
As a security measure, I will not run automated tests on PRs that are not from white-listed contributors.

In order to allow automated tests to run, please ask one of the project maintainers to review the code and then do one of the following:

1. Type `ci test please` on this PR to trigger automated tests for it.
2. Type `ci add to whitelist` on this PR to trigger automated tests for it and also add you to the contributor white-list so that your future PRs will be tested automatically.
3. If you are planning to contribute to more than one project, maybe it's better to ask them to add you to the project organization, so you'll be able to run tests for all the organization's projects.

File: README.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ QCOW variables:
 
 | Name          | Default value                                            |  Description                                                 |
 |---------------|----------------------------------------------------------|--------------------------------------------------------------|
-| miq_qcow_url  | http:////releases.manageiq.org/manageiq-ovirt-fine-1.qc2 | The URL of the ManageIQ QCOW image. |
+| miq_qcow_url  | http://releases.manageiq.org/manageiq-ovirt-fine-1.qc2 | The URL of the ManageIQ QCOW image. |
```
Comment (mwperina) [inline README.md]: Not relevant in this patch, but shouldn't we change the URL in the doc to Gaprindashvili release?
@machacekondra

**Review by mwperina — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (machacekondra) [reply README.md]: Yep, can you please change the URL to : http://releases.manageiq.org/manageiq-ovirt-gaprindashvili-2.qc2

Comment (pkubica) [reply README.md]: Done.

**Review by mwperina — APPROVED** [review]

**Pull request merged**